lib/gooddata/models/segment.rb

```diff
@@ -73,7 +74,7 @@ def create(data = {}, options = {})
         segment_id = data[:segment_id]
         fail 'segment_id has to be provided' if segment_id.blank?
         client = options[:client]
-        segment = client.create(GoodData::Segment, GoodData::Helpers.stringify_keys(SEGMENT_TEMPLATE).merge('domain' => options[:domain]))
+        segment = client.create(GoodData::Segment, GoodData::Helpers.stringify_keys(SEGMENT_TEMPLATE), domain: options[:domain])
```

Comment:
Refers to 'options' more than self (maybe move it to another class?) (FeatureEnvy)